contracts/testBytesJobSpec.sol

```diff
@@ -0,0 +1,48 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.7;
+
+import "@chainlink/contracts/src/v0.8/ChainlinkClient.sol";
+
+contract RequestGetBytes is ChainlinkClient {
+  using Chainlink for Chainlink.Request;
+
+  // variable bytes returned in a single oracle response
+  bytes public data;
+  string public image_url;
+
+  constructor(
+  ) {
+    setChainlinkToken(LINK_TOKEN_ADDRESS);
+    setChainlinkOracle(OPERATOR_ADDRESS);
+  }
+
+  function requestBytes(
+  )
+    public
+  {
+    bytes32 specId = "JOB_ID";
```

Comment:
Should we move `specId` into the params list for `requestBytes(specId)` - that way we can make it dynamic post deployment of testBytesJobSpec.sol and call it with a `jobId` from the `requestBytes.py` script.